src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -65,6 +128,24 @@ protected override async Task<bool> IsSemanticTriggerCharacterAsync(Document doc
             return token.Kind() != SyntaxKind.NumericLiteralToken;
         }
 
+        private async Task<bool?> IsTriggerInArgumentListAsync(Document document, int characterPosition, CancellationToken cancellationToken)
+        {
+            var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+            if (!CompletionUtilities.IsArgumentListCharacter(text[characterPosition]))
+            {
+                return null;
+            }
+
+            var tree = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var token = tree.FindToken(characterPosition);
+            if (token.Parent.IsKind(SyntaxKind.ArgumentList))
```

Comment:
is this supposed to be *only* for arg lists for now?  what about indexers, attributes, etc.?

Reply:
Added the other relevant syntaxkinds